ui/drivers/qt/gridview.cpp

```diff
@@ -332,7 +339,9 @@ void GridView::setSelection(const QRect &rect, QFlags<QItemSelectionModel::Selec
 QRegion GridView::visualRegionForSelection(const QItemSelection &selection) const
 {
    QRegion region;
-   foreach(const QItemSelectionRange &range, selection)
+   QItemSelectionRange range;
+
+   foreach(range, selection)
```

Comment:
Just FYI for in the future, we try to avoid `foreach` because it makes a copy of all the items, and it is scheduled for deprecation. See [here](https://www.kdab.com/goodbye-q_foreach/) and [here](https://codereview.qt-project.org/#/c/147363/).